File: dynamicpdf-vbnet-examples/Examples/Data/PlaceHolderExampleData.vb
Imports System.Collections.Generic

Public Class PlaceHolderExampleData

    Public Class Website
        Public Property Name As String
        Public Property Description As String
        Public Property Url As String
    End Class

    Private Shared websites_1 As New List(Of Website) From {
        New Website With {.Name = "DynamicPDF", .Description = "Founded in 2002, DynamicPDF is a leader in PDF SDKs and libraries.", .Url = "https://www.dynamicpdf.com"},
        New Website With {.Name = "Facebook", .Description = "Founded in 2004, Facebook is a popular social networking website.", .Url = "https://www.facebook.com"},
        New Website With {.Name = "Google", .Description = "Founded in 1997, Google is a popular search engine.", .Url = "https://www.google.com"},
        New Website With {.Name = "Twitter", .Description = "Founded in 2006, Twitter is a popular news and social media website.", .Url = "https://twitter.com"}
    }

    Public Shared ReadOnly Property Websites As List(Of Website)
        Get
            Return websites_1
        End Get
    End Property

End Class

File: dynamicpdf-csharp-examples/Examples/data/PlaceHolderExampleData.cs
using System.Collections.Generic;

/// <summary>
/// Summary description for PlaceHolderExampleData
/// </summary>
public class PlaceHolderExampleData
{
    public class Website
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public string Url { get; set; }
    }

    private static List<Website> websites = new List<Website>()
        {
            new Website { Name = "DynamicPDF", Description = "Founded in 2002, DynamicPDF is a leader in PDF SDKs and libraries.", Url = "https://www.dynamicpdf.com" },
            new Website { Name = "Facebook", Description = "Founded in 2004, Facebook is a popular social networking website.", Url = "https://www.facebook.com" },
            new Website { Name = "Google", Description = "Founded in 1997, Google is a popular search engine.", Url = "https://www.google.com" },
            new Website { Name = "Twitter", Description = "Founded in 2006, Twitter is a popular news and social media website.", Url = "https://twitter.com" }
        };
    public static List<Website> Websites
    {
        get { return websites; }
    }
}
